website/content/faqs/what-is-enigma-and-how-does-it-determine-variant-classifications.md

```diff
@@ -1,6 +1,6 @@
-ENIGMA \(Evidence-based Network for the Interpretation of Germline Mutant Alleles\) is an international consortium of researchers who provide expert opinions on the clinical significance of BRCA1, BRCA2, and other breast cancer genes. This consortium serves as the BRCA expert review body of ClinGen. ENIGMA uses a multi-factorial analysis that weighs together many types of variant data, such as population frequencies, protein and transcript structure, and epidemiological measures. The analysis is tuned specifically for the BRCA genes, and subsumes IARC criteria alongside other statistical analyses developed by ENIGMA collaborators.
+ENIGMA (Evidence-based Network for the Interpretation of Germline Mutant Alleles) is an international consortium of researchers who provide expert opinions on the clinical significance of variation in BRCA1, BRCA2, and other breast cancer genes. This consortium serves as the BRCA expert review body of ClinGen. ENIGMA uses classification criteria tuned specifically for the BRCA genes to evaluate the clinical significance of BRCA1/2 variants. Evidence reviewed includes many different types of data, including population frequencies, clinical information, and epidemiological measures. This data also includes effects of variation on transcript and protein structure and function.  All available information is combined to estimate the probability that a variant is cancer-causing, using qualitative (rules-based) and quantitative (statistical) method multi-factorial likelihood analysis.
```

Comment:
I noticed some sentences have one space after a period and others have 2 spaces after a period. It seems like this should be more consistent.

E.G.
`review body of ClinGen. ENIGMA uses`
vs
`function.  All available `

Reply:
Thanks for bringing the double spaces to my attention. I will attend to them now, and re-push ASAP.